scraper.py

```diff
@@ -5,31 +5,39 @@
 import csv
 import sys
 import mysql.connector
+from plyer import notification
 
-c = sys.argv[1]
-yr = sys.argv[2]
+c = sys.argv[0]
+yr = sys.argv[0]
```

Comment:
same thing carries over here about sys.argv. This is the order of arguments, not the range of the variable